code/game/objects/structures/janicart.dm

```diff
@@ -165,6 +165,7 @@
 	icon_state = "pussywagon"
 	anchored = 0
 	density = 1
+	pressure_resistance = 5

```

Comment:
You might as well just set this on the base chair object instead of setting it per object.